### ISSUE TITLE
PEOPLE: Mitglieder verify page Werbebild

### ISSUE THREAD

**Issue by daniel-illi** [hitobito/hitobito_sac_cas#720]
Mit #531 wurde die "verify page" für den SAC angepasst und dabei ein Platzhalterbild für Werbung eingebaut.
Dieses soll nun durch die realen Werbebilder ersetzt werden. Dabei soll das angezeigte Bild abhängig sein von der eingestellten UI Sprache.

# Mockup

hier folgt ein Mockup ...

# Tech-Spec

- sinngebende Zielrichtung
- besondere/bekannte Bedingungen und Grenzfälle
- technische Einschränkungen
- Ein- und Ausgabeformate beschreiben
- Umsetzungsplan ergänzen oder anpassen

# ToDo

- [ ] Migration erstellen
- [ ] Domainklasse erstellen oder anpassen
- [ ] Model/View/Controller anpassen
- [ ] Neues Feature mit FeatureGate deaktivierbar machen
- [ ] Specs schreiben
- [ ] Kunde wegen Übersetzungen informieren
- [ ] Mit angemessener Rolle "durchklicken"
- [ ] CHANGELOG-Eintrag unter "unreleased" unten hinzufügen

**Comment by Largo** [comment]
@daniel-illi Eventuell noch mit besserer qualitatiaet ersetzen. Ich empfehle .png.

**Comment by tobiasstern** [comment]
@Largo die verify page wird bei mir immer in Deutsch angezeigt, auch wenn ich eine andere Sprache gewählt habe im UI. Ist da noch ein Fehler?

**Comment by Largo** [comment]
@tobiasstern Es gibt dort noch keine Übersetzungen. Kann es das sein?  Oder ändert sich das Bild und die Logos ebenfalls nicht?
Ebenfalls hat die Webadresse keinen Sprachteil drin, daher nimmt es das aus den Cookies. Da weiss ich nicht, warum das so gemacht wurde. Vermutlich hat das einen Grund z.b. kleineren QR Code. Beim PDF wird inzwischen die gespeicherte Sprache der Person genommen.

**Comment by tobiasstern** [comment]
Ja, die Logos waren immer die deutschen, darum ist es mir aufgefallen. Aber ich denke, es ist okay so.

- Ich dachte zuerst, wenn ich Hitobito auf FR umstelle auf meinem PC, dann sollte die verify page auf dem handy auch FR sein. Aber macht Sinn, dass dies nicht übernommen wird, wenn es nirgends übergeben wird.
- Wenn ich auf meinem Handy im Hitobito auf FR umstelle und dann den QR Code vom DE-Hitobito auf dem PC scanne, wird die verify korrekt in FR angezeigt 👍
